views/buyerlogbook.ejs

```diff
@@ -141,7 +61,7 @@
     
     <% include ./partials/notificationbar %>
     <div class="col text-right">
-      <select class="custom-select d-block w-100 " name="produce_filter" id="produce_filter" required>
+      <select onchange="activateProduceFilter()" class="custom-select d-block w-100 " name="produce_filter" id="produce_filter" required>
```

Comment:
where is the activateProduceFilter function defined?

Reply:
It is defined above the FilterCheck() function - I just made it work with onChange instead of the search button that was there previously.